ci-operator/config/openshift/os/openshift-os-master.yaml

```diff
@@ -110,9 +115,8 @@ tests:
         cd "$COSA_DIR"
         # Tags with the cosa build ID / arch
         cosa push-container registry.ci.openshift.org/rhcos-devel/rhel-coreos
-
         cosa push-container registry.ci.openshift.org/rhcos-devel/rhel-coreos:latest
```

Comment:
Is there then a way to only push `:latest` if the branch name is `master`?

Reply:
Thanks for the reference! I was basing this on what was in the `$JOB_SPEC` env var. I didn't see the ones you referenced in the build pod, but if they're there, great! I'll do a bit of experimentation first since I don't know what's actually there. I would much prefer to use the env vars you referenced rather than decoding that JSON payload.

As for what you described about pushing to `:latest` only on `master`, that should be possible. If I've got a branch name, I can put a conditional on it.